climada/engine/unsequa/calc_impact.py

```diff
@@ -205,7 +205,7 @@ def uncertainty(self,
         with log_level(level='ERROR', name_prefix='climada'):
             if pool:
                 LOGGER.info('Using %s CPUs.', pool.ncpus)
-                chunksize = min(unc_sample.n_samples // pool.ncpus, 100)
+                chunksize = max(min(unc_sample.n_samples // pool.ncpus, 100), 1)
```

Comment:
```suggestion
                chunksize = min(unc_sample.n_samples // pool.ncpus + 1, 100)
```
I think this is more elegant.

Reply:
Hm. Maybe. But when you have 6 samples and 3 cpus the chunksize is 3 where 2 would fit better- right?
Doesn't matter I guess, but the elegance comes at a price of reduced readability imho.